CONTRIBUTING.md

```diff
@@ -163,22 +201,42 @@ $ git lfs pull
 
 [git-lfs]: https://git-lfs.github.com
 
-## What to work on
+## Issues
+
+The [GitHub tracker][issu] is our canonical source of truth around issues,
+bugs, performance problems, feature requests, and so on.  If you encounter any
+issues when developing on urbit, feel free to submit a report about it here.
```

Comment:
```suggestion
issues when developing on Urbit, feel free to submit a report about it here.
```

Reply:
We settled this internally some time ago. Urbit is the project/product. An urbit is an individual instance.